cb_medical_diagnostic_report/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Cb Medical Diagnostic Report",
+    "summary": """Allows the creation of medical diagnostic reports""",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Creu Blanca",
+    "website": "https://github.com/tegin/medical-fhir",
+    "depends": ["medical_diagnostic_report", "medical_administration"],
```

Comment:
Maybe we can integrate the signature storage here :smile: 
medical_signature_storage